docs/sql-pyspark-pandas-with-arrow.md

```diff
@@ -65,132 +65,188 @@ Spark will fall back to create the DataFrame without Arrow.
 
 ## Pandas UDFs (a.k.a. Vectorized UDFs)
```

Comment:
Please see also the PR description of https://github.com/apache/spark/pull/27165#issue-361321870